packages/battery_plus/battery_plus/lib/battery_plus.dart

```diff
@@ -37,7 +39,7 @@ class Battery {
 
   /// check if device is on battery save mode
   ///
-  /// Currently only impemented on Android, IOS and Windows.
+  /// Currently only implemented on Android, IOS and Windows.
```

Comment:
Since we're fixing typos, `iOS` would be better

Reply:
I couldn't let it go, been seeing the typo the whole day :joy:

Reply:
That's the spirit!